MarsOnboarding24/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,157 @@
+using MarsOnboarding24.Pages;
+using MarsOnboarding24.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using TechTalk.SpecFlow;
+
+namespace MarsOnboarding24.StepDefinitions
+{
+    [Binding]
+    public class LanguageStepDefinitions : CommonDriver
+    {
+        LoginPage loginPageObj = new LoginPage();
```

Comment:
please make use of constructors 
https://www.tutorialspoint.com/csharp/csharp_classes.htm
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors